dynamic/README.md

```diff
@@ -1,19 +1,38 @@
-Instana Agent Docker
-====================
+# Instana Agent Dynamic Docker Image
 
-This build of the instana agent includes requires access to the publicly hosted Instana maven repository in order to download sensors. It requires proxy settings for egress access to the ${INSTANA_AGENT_ENDPOINT}, which may either be for your self hosted Instana installation or for the Instana SaaS, and for access to the Instana maven repository.
+This build of the Instana agent requires access to the publicly hosted Instana maven repository in order to download sensors.
+It may require proxy settings for egress access to:
 
-Building
-========
+* the `${INSTANA_AGENT_ENDPOINT}`, which may either be for your self-hosted Instana installation or for the Instana SaaS
+* the [Instana Artifactory repository](https://artifact-public.instana.io/), unless you set up a mirror and use the `INSTANA_MVN_REPOSITORY_URL` to direct the Instana Agent to use it
 
-docker build ./ --build-arg FTP_PROXY=${INSTANA_AGENT_KEY} --no-cache
+Additional documentation about the usage of this image is available on the [Installing the Host Agent on Docker](https://www.instana.com/docs/setup_and_manage/host_agent/on/docker) documentation.
 
-*Note*
+## Building
 
-FTP_PROXY is being abused to pass in the agent key for the package download during docker build, we are doing this until docker build time secrets issue is resolved: [issue GH33343](https://github.com/moby/moby/issues/33343)
+Needs Docker 18.09 or higher:
 
-Docker Hub
-==========
+```sh
+export TARGETPLATFORM=linux/s390x && docker buildx build --no-cache --build-arg DOWNLOAD_KEY=nqtbV5cEQ5ev0MFzOIwskg "--platform=${TARGETPLATFORM}" --build-arg "TARGETPLATFORM=${TARGETPLATFORM}" . -t instana/agent
```

Comment:
!!! Should this `DOWNLOAD_KEY` not be some dummy or placeholder... ???

Reply:
Also the rest of this command does not appear to be easily copy-pastable. It has some issues.